stytch/b2b/sessions.go

```diff
@@ -323,10 +336,27 @@ func (c *SessionsClient) AuthenticateJWTLocal(
 		return nil, sessions.ErrJWTTooOld
 	}
 
-	return marshalJWTIntoSession(claims)
+	resp, err := marshalJWTIntoSession(claims)
+	if err != nil {
+		return nil, err
+	}
+
+	cachedPolicy, _ := c.LocalCache.Get("policy")
```

Comment:
```go

type PolicyCache struct {
    rbac *rbac.Client
    policy *rbac.Policy
    lastUpdatedAt time.Time
}

func NewPolicyCache(rbac *rbac.Client) *PolicyCache {
    return &PolicyCache{rbac: rbac}
}

func (pc *PolicyCache) Get() *rbac.Policy {
    if pc.policy == nil || time.Now() - lastUpdatedAt > 300 * time.Second {
        // refresh
        pc.policy = rbac.Policy().Policy
        pc.lastUpdatedAt = time.Now()
    }
    return pc.policy
}
```


In codegen you'll have to add logic like:
```
{% if rbac_service %}
    policyCache = stytch.NewPolicyCache(b2b.NewRBACClient(a.client))
{% endif %}
```

And also logic so that policyCache gets passed down to every service that needs it